BaseApi/Transformer/AbstractTransformer.php

```
@@ -82,4 +92,24 @@ public function transform($mixed)
     public function reverseTransform(FacadeInterface $facade, $source = null)
     {
     }
+
+    /**
+     * @throws TransformerParameterTypeException
+     *
+     * @return FacadeInterface
+     */
+    protected function newFacade()
+    {
+        if (null === $this->facadeClass) {
+            throw new TransformerParameterTypeException();
```

you should throw a specific exception for this error